index.test.js

```diff
@@ -123,3 +123,117 @@ describe('#replace', () => {
     });
   });
 });
+
+describe('#increment', () => {
+  it('should export a increment function', () => {
+    expect(m.increment).toBeInstanceOf(Function);
+  });
+
+  it('should handle empty input', () => {
+    const fieldsMap = {};
+    const result = m.increment(fieldsMap);
+
+    expect(result).toEqual({
+      lang: 'painless',
+      source: '',
+      params: {}
+    });
+  });
+
+  it('should return a script to set 2 simple fields', () => {
```

Comment:
> to set 2 simple fields

Looks like smth copy-pasted from `#set` tests 😅 

I'd change the test title to `to increment 2 simple fields` / `to multiply 2 simple fields` to better convey the meaning

Reply:
It is definitely pasted from #set.

titles will be fixed.